stdlib/private/StdlibUnittest/StdlibUnittest.swift

```diff
@@ -103,6 +103,22 @@ public struct SourceLocStack {
   }
 }
 
+fileprivate struct AtomicBool {
+    
+    private var _value = _stdlib_AtomicInt
```

Comment:
The previous build uncovered this typo:

```suggestion
    private var _value: _stdlib_AtomicInt
```